script.js

```diff
@@ -0,0 +1,75 @@
+const bookList = document.getElementById('book-list');
+const addBook = document.getElementById('add-book');
+const newTitle = document.getElementById('new-title');
+const newAuthor = document.getElementById('new-author');
+let bookData = [];
+class Book {
+  constructor(title = 'book', author = 'people', id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+}
```

Comment:
- [ ] Please use Javascript objects to create book data, You will refactor this code and use class and instances in the next project.